src/main/java/hudson/plugins/jacoco/JacocoPublisher.java

```diff
@@ -451,13 +461,18 @@ public void perform(@Nonnull Run<?, ?> run, @Nonnull FilePath filePath, @Nonnull
         logger.print("\n[JaCoCo plugin] Saving matched class directories for class-pattern: " + classPattern + ": ");
         for (FilePath file : matchedClassDirs) {
             dir.saveClassesFrom(file);
-            logger.print(" " + file);
```

Comment:
These log-outputs were done on purpose to help with investigation of problems, any reason why you removed them as part of this PR?